src/Room.src.json

```diff
@@ -0,0 +1,43 @@
+{
+    "type": "Room",
+    "uri": "http://schema4i.org/Room",
+    "description": "A room is a distinguishable space within a structure, usually separated from other spaces by interior walls.",
+    "links": [{
+        "url": "http://schema.org/Room",
+        "description": "Original Schema.org type"
+    }],
+    "parents": [
+        
+    ],
+    "base": [{ "@id": "http://schema4i.org/Accommodation" }],
+    "multipletypes": {
+        "AccommodationCategory": [
+            { "@id": "http://schema.org/Text" },
+            { "@id": "http://schema4i.org/EnumRoomTypeCode" }
+        ]
+    },
+    "context": {
+        "@context": {
+            "@version": 1.1,
+            "s4i": "http://schema4i.org/",
+            "schema": "http://schema.org/",
+            "Room": "schema:Room",
+            "AccommodationCategory": {
+                "@id": "schema:accommodationCategory"
+            }
+        }
+    },
+    "playground": [{
+        "title": "A maximum Accommodation",
+        "tab": "tab-expanded",
+        "input": {
+            "@context": [
+                "http://schema4i.org/Room",
+                "http://schema4i.org/EnumDamageCauseCode.jsonld"
+            ],
```

Comment:
At this point a type declaration is needed, because `AccommodationCategory` is a multitype attribute. To let a client know which type is used we need a `@context` for it. We fixed this on our own. Please see changes.